Formula/trurl.rb

```diff
@@ -18,6 +18,10 @@ class Trurl < Formula
 
   uses_from_macos "curl", since: :ventura # uses CURLUE_NO_ZONEID, available since curl 7.81.0
 
+  on_system :linux, macos: :monterey_or_older do
+    depends_on "curl" => [:build, ">= 7.81.0"]
+  end
```

Comment:
Setting `>= 7.81.0` doesn't do anything. This is also already implied by
```ruby
uses_from_macos "curl", since: :ventura
```
```suggestion
```

Reply:
Sorry if this is a stupid question, but what if those linux system or macos older than monterey installed curl already? Thought that they'll still satisfy the condition?

Reply:
`uses_from_macos "curl", since: :ventura` means `depends_on "curl"` on Linux and on macOS versions older than Ventura.

`depends_on "curl"` means it depends on Homebrew-built `curl`, which can be no older than 8.0.1, which was the [version of `curl`](https://github.com/Homebrew/homebrew-core/blob/c25cea343e862c1d8a690d49c4a3186a8be47d1a/Formula/curl.rb#L4) when `trurl` was [added to Homebrew/core](https://github.com/Homebrew/homebrew-core/commit/c25cea343e862c1d8a690d49c4a3186a8be47d1a).

Reply:
@carlocab just want to be sure, will `depends_on` make sure install the latest `curl` version, so even an old Homebrew-built `curl` was installed, it will be upgraded to v8.0.1? Thanks.

Reply:
Yes.